draft-ietf-quic-recovery.md

```diff
@@ -245,6 +245,13 @@ QUIC specifies a time-based definition to ensure one or more packets are sent
 prior to a dramatic decrease in congestion window; see
 {{persistent-congestion}}.
 
+### Increase the min congestion window to 2 packets
+
+QUIC recommends a minimum congestion window of 2 packets instead of TCP's 1.
+2 packets avoid waiting for a delayed acknowledgement and allow the PTO to
+send 2 packets instead of 1, which can be particularly important during the
```

Comment:
The PTO is not limited by the cwnd.

Reply:
What Marten said.  This justification makes no sense.  Probing with two packets is different.  

As far as I can tell, this is an arbitrary choice, driven primarily by a desire to avoid the performance floor being too low.

Reply:
Always hitting delayed acks can be a major problem as well, though QUIC allows explicitly communicating those, which avoids some problems.

Reply:
I see this as a more fundamental deviation to TCP than other changes, because although it will have some performance benefit in common cases, it also changes the flow aggressiveness in cases of extreme overload. As I udnerstand, the basis of TCP's back-off is that it scales back cwnd to 1 MSS, and then should scale back in the interval between sending 1 MSS (fractional MSS). If QUIC has a minimum of 2*MSS in QUIC, does it immediately change to 2*RTT interval, otherwise this appears to be significantly more aggressive flow under congestion collapse?

Reply:
I'd expect congestion collapse to be dominated by the PTO response, not the CWND.

I'm not aware of efforts to do fractional MSS in common use.  Do you have a reference for that?

Reply:
Do we agree on what is intended by congestion-collapse, i.e. persistent long-lasting high level of congestion, as oppose to just "congestion"? (e.g. when there is a very low probability of a successful ACK). ... In will find something on reducing the MSS below once/RTT.